README.md

```diff
@@ -13,19 +13,26 @@ located.
 const path = require('path');
 
 const nconf = require('nconf');
-require('./lib/nconfetti');
 
-const path = path.resolve(__dirname, 'tests/configs/env');
+require('./../index.js'); // replace with require('nconfetti');
 
 nconf.argv()
   .env('__');
 
-nconf.use('Nconfetti', {path: path, env: process.env.NODE_ENV});
+nconf.use('Nconfetti', {path: path.resolve(__dirname, '../tests/configs/without_env')});
 
-console.log(nconf.get('development_config:development_config'));
+console.log(nconf.get('simple_config:entry'));
+```
 
+## Try a sample
 
-```
+- see sample/simple.js
+- runit with
+  - npm install
+  - node samples/simple.js
+ - have fun
+
+## Description
 
 After require `nconfetti` the backend will be added as storage backend
```

Comment:
Finde ich schlecht verständlich - besser: `After requiring nconfetti the its storage will automagically be registered to nconf.`